testsuite/features/step_definitions/command_steps.rb

```diff
@@ -1515,6 +1505,10 @@
                    echo '#{server_node.public_ip} #{server_node.full_hostname} #{old_hostname}' >> /etc/hosts &&
                    echo '#{server_node.public_ip} #{new_hostname}#{server_node.full_hostname.delete_prefix(server_node.hostname)} #{new_hostname}' >> /etc/hosts")
   get_target('server', refresh: true) # This will refresh the attributes of this node
+  hostname, _result = get_target('server').run("hostname")
+  hostname.strip!
+
+  raise "Wrong hostname after changing it. Is: #{hostname}, should be: #{new_hostname}" unless hostname == new_hostname
```

Comment:
Maybe such basic checks would deserve a `Then` step of their own. It would make more explicit / readable that we do them.

Reply:
Yes, we could do that. However, my main issue with this is, that our hostname rename and then changing the hostname back step, has a hardcoded new hostname variable in it. We do not define this new hostname in the step itself. Which is good, since across all our CIs and PR tests, the prefix of the hostname is different. Those are the reasons why I did not implement a new step of its own to check for the new hostname.